vpdq/python/vpdq.pyx

```diff
@@ -78,14 +76,14 @@ def hash_to_hex(hash_value: "Hash256"):
     Returns:
         str: hex str of hash
     """
-    return hashToString(hash_value)
```

Comment:
blocking: I am very confused - the signature of this function says it is returning a string. What is it returning instead, and why don't we fix it at https://github.com/facebook/ThreatExchange/blob/main/vpdq/python/vpdq.pyx#L21 ?

Reply:
It is returning the c_string. The string type is from from ``` libcpp.string cimport string ``` With this cdef extern, we cannot modify codes here. It is only extracting the c function. Have to wrap it around with a python function and solve the problem there.

Reply:
blocking: I believe there should be a solution that prevents the unencoded from leaving your python bindings, where they will later confuse folks trying to use your library.

Does either of these match what you are observing? 
1. https://stackoverflow.com/questions/12500069/ctypes-how-to-pass-string-from-python-to-c-function-and-how-to-return-string
2. https://pybind11.readthedocs.io/en/stable/advanced/cast/strings.html

Reply:
The current output from the vpdq hash_to_hex is a binary string literal. For example, an hash "000ffff...." will be "b'000ffff.....". After this pr, the hash_to_hex will return "000ffff....".